rust/operator-binary/src/controller.rs

```diff
@@ -97,6 +97,13 @@ const USER_INFO_FETCHER_CREDENTIALS_DIR: &str = "/stackable/credentials";
 
 const DOCKER_IMAGE_BASE_NAME: &str = "opa";
 
+// logging defaults
+const DECISION_LOGGING_ENABLED: bool = false;
+const FILE_LOG_LEVEL: LogLevel = LogLevel::INFO;
+const CONSOLE_LOG_LEVEL: LogLevel = LogLevel::INFO;
+const SERVER_LOG_LEVEL: LogLevel = LogLevel::INFO;
+const DECISION_LOG_LEVEL: LogLevel = LogLevel::NONE;
```

Comment:
suggestion: Prefix the constant names with `DEFAULT_` because they define default values.

```suggestion
// logging defaults
const DEFAULT_DECISION_LOGGING_ENABLED: bool = false;
const DEFAULT_FILE_LOG_LEVEL: LogLevel = LogLevel::INFO;
const DEFAULT_CONSOLE_LOG_LEVEL: LogLevel = LogLevel::INFO;
const DEFAULT_SERVER_LOG_LEVEL: LogLevel = LogLevel::INFO;
const DEFAULT_DECISION_LOG_LEVEL: LogLevel = LogLevel::NONE;
```